property.go

```diff
@@ -108,3 +133,82 @@ func (client *Client) DeletePropertyDefinition(input string) error {
 	err := client.Mutate(&m, v, WithName("PropertyDefinitionDelete"))
 	return HandleErrors(err, m.Payload.Errors)
 }
+
+func (client *Client) GetProperty(owner string, definition string) (*Property, error) {
+	var q struct {
+		Account struct {
+			Property Property `graphql:"property(owner: $owner, definition: $definition)"`
+		}
+	}
+	v := PayloadVariables{
+		"owner":      *NewIdentifier(owner),
+		"definition": *NewIdentifier(definition),
+	}
+	err := client.Query(&q, v, WithName("PropertyGet"))
+	if q.Account.Property.Definition.Id == "" {
+		err = fmt.Errorf("Property with ID or alias matching '%s' on Service with ID or alias matching '%s' not found", definition, owner)
+	}
+	return &q.Account.Property, HandleErrors(err, nil)
+}
+
+func (client *Client) PropertyAssign(input PropertyInput) (*Property, error) {
+	var m struct {
+		Payload struct {
+			Property Property         `graphql:"property"`
+			Errors   []OpsLevelErrors `graphql:"errors"`
+		} `graphql:"propertyAssign(input: $input)"`
+	}
+	v := PayloadVariables{
+		"input": input,
+	}
+	err := client.Mutate(&m, v, WithName("PropertyAssign"))
+	return &m.Payload.Property, HandleErrors(err, m.Payload.Errors)
+}
+
+func (client *Client) PropertyUnassign(owner string, definition string) error {
+	var m struct {
+		Payload struct {
+			Errors []OpsLevelErrors `graphql:"errors"`
+		} `graphql:"propertyUnassign(owner: $owner, definition: $definition)"`
+	}
+	v := PayloadVariables{
+		"owner":      *NewIdentifier(owner),
+		"definition": *NewIdentifier(definition),
+	}
+	err := client.Mutate(&m, v, WithName("PropertyUnassign"))
+	return HandleErrors(err, m.Payload.Errors)
+}
+
+func (service *Service) GetProperties(client *Client, variables *PayloadVariables) (*ServicePropertiesConnection, error) {
+	var q struct {
+		Account struct {
+			Service struct {
+				Properties ServicePropertiesConnection `graphql:"properties(after: $after, first: $first)"`
+			} `graphql:"service(id: $service)"`
+		}
+	}
+
+	if service.Id == "" {
+		return nil, fmt.Errorf("Unable to get properties, invalid Service id: '%s'", service.Id)
+	}
+	if variables == nil {
+		variables = client.InitialPageVariablesPointer()
+	}
+	(*variables)["service"] = service.Id
+	if err := client.Query(&q, *variables, WithName("ServicePropertiesList")); err != nil {
+		return nil, err
+	}
+	if service.Properties == nil {
+		service.Properties = &ServicePropertiesConnection{}
+	}
+	service.Properties.Nodes = append(service.Properties.Nodes, q.Account.Service.Properties.Nodes...)
+	service.Properties.PageInfo = q.Account.Service.Properties.PageInfo
+	for service.Properties.PageInfo.HasNextPage {
+		(*variables)["after"] = service.Properties.PageInfo.End
+		_, err := service.GetProperties(client, variables)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return service.Properties, nil
```

Comment:
I think we are missing calculating `TotalCount` here

Reply:
Added code to update `TotalCount` in the latest commit.